src/library-authoring/components/ContainerCard.tsx

```diff
@@ -147,7 +144,7 @@ const ContainerCardPreview = ({ containerId, showMaxChildren = 5 }: ContainerCar
           }
           return (
             <div
-              key={`container-card-preview-block-${id}`}
+              key={`container-card-preview-block-${usageKey}`}
```

Comment:
React keys must be unique only on the same loop (not app/component-wise), so we are fine with `key={usageKey}` here.

Also, we can have the same component multiple times.
```suggestion
               // A container can have multiple instances of the same block
              // eslint-disable-next-line react/no-array-index-key
              key={`${usageKey}-${idx}`}
```
PS: I'm fixing this on https://github.com/openedx/frontend-app-authoring/pull/1868